### DISCUSSION TITLE
Tool 'coverlet.console' failed to install

### DISCUSSION THREAD

**Discussion by TerryChan2** [coverlet-coverage/coverlet#1493] [Q&A]
Hi,

I use coverlet .console in cake build and already defined this **#tool dotnet:?package=coverlet.console&version=6.0.0**
The script can be executed successfully most of the time but occasionally it would encountered below error:

_The tool package could not be restored.
error	Tool 'coverlet.console' failed to install. This failure may have been caused by:
error	* You are attempting to install a preview release and did not use the --version option to specify the version.
error	* A package by this name was found, but it was not a .NET tool.
error	* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
error	* You mistyped the name of the tool.
error	For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool
build	dotnet exited with 1
build	Could not find any relevant files for tool 'coverlet.console'. Perhaps you need an include parameter?
error	Error: Failed to install tool 'coverlet.console'._

If I rerun the script again, it could be successfully executed.  Any idea about this issue?

**Comment by Bertk** [comment]
Sorry, we have no experience with cake.
